### PR TITLE
Install linux command

### LINKED ISSUE #724: rework the downloads page by OS
and document installing ziti CLI Linux package

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

**Comment by qrkourier** [comment]
@dovholuknf Preview is updated with query strings like `?os=Windows` and https://ziti-doc-git-install-linux-command-openziti.vercel.app/docs/downloads?os=macOS, but I'm still figuring out how to detect device OS.

Also, the preview sometimes shows the ToC despite `hide_table_of_contents: true` for some unknown reason. Will revisit tomorrow.

I plan to rename the Linux packages based on whatever outcome of discussion in https://github.com/openziti/ziti/issues/911#issuecomment-1747085431 before merging this doc to avoid publishing something that's imminently obsolete anyway.

**Review by dovholuknf — DISMISSED** [review]

**Comment by dovholuknf** [comment]
looks great!

**Comment by dovholuknf** [comment]
actually hold up... 
image

**Comment by qrkourier** [comment]
> actually hold up...

What's the hold up? BTW, merging is on hold anyway until the packages documented herein are released, and because the table of contents is still showing.

**Comment by dovholuknf** [comment]
can you remove all the `##` from the doc and replace with <p> tags or anything else to 'fix' that sidebar?

**Comment by qrkourier** [comment]
That's the table of contents, and I should be able to adjust the heading levels and minimum heading level for ToC, but it shouldn't appear at all. That's what @JeremyTellier is investigating.

The front matter has:

```yaml
hide_table_of_contents: true
```

**Comment by qrkourier** [comment]
> can you remove all the `##` from the doc and replace with

Now with moar tinier headings and no floating sidebar :tada: 

https://ziti-14dwrkmrk-openziti.vercel.app/docs/downloads

**Comment by dovholuknf** [comment]
it'd be nice if all the tabs behave the same. right now some tabs are wide, some are not.

**Comment by qrkourier** [comment]
> it'd be nice if all the tabs behave the same. right now some tabs are wide, some are not.

I hadn't noticed. That's more styling finesse than I'm eager to tackle. It would have to be a stylesheet customization that sets a minimum width for the generated divs. I'm not sure that would be a good thing for a responsive site, especially on mobile where the user must swipe horizontally to scroll to the desired tab. Adding width adds centimeters of whitespace they have to trudge through.

**Comment by qrkourier** [comment]
We should add the Play and App Store buttons instead of plain text links. They're easier to see and touch on mobile.

**Comment by qrkourier** [comment]
@dovholuknf I'm at a stopping point and ready for the next review round on this one. The main difference from the last round is updating the Linux package repo details to match the changes for `ziti` and `ziti-edge-tunnel` package deployments.

File: docusaurus/src/theme/OsTabs/index.js
Comment (dovholuknf) [inline docusaurus/src/theme/OsTabs/index.js:23]: put the brackets inside the comments? maybe remove the whole thing?

**Review by dovholuknf — APPROVED** [review]
looks good. i'd really like to see some better differentiation on the linux downloads though. it's not easy to see the 'family' choices. A "pick the linux family" above the tabs would maybe help
image

**Comment by qrkourier** [comment]
New PR for refinements: https://github.com/openziti/ziti-doc/pull/738

**Pull request merged**
